web/src/components/software/ProductSelectionPage.jsx

```diff
@@ -88,16 +89,18 @@ function ProductSelectionPage() {
 
   return (
     <>
-      <Title>Product selection</Title>
+      {/* TRANSLATORS: page header */}
+      <Title>{_("Product selection")}</Title>
```

Comment:
not sure if in title it should not be all capitalized like `Product Selection`

Reply:
Why? Do we have any guidelines about that?